### DISCUSSION TITLE
Suggestion: stop feeding all servers feed (/public) when user is scrolling down

### DISCUSSION THREAD

**Discussion by and-rea** [mastodon/mastodon#27357] [General]
I noticed the page continues to receive feeds while I am scrolling down to read the past ones.

I think that the live feed must be running only when I am at the top of the page, this also prevents bandwidth waste when the page is left unattended.

I wonder if the browser would explode on resource usage if I left it unattended all night 😝

**Comment by StrangeMinds1960** [comment]
Preferences > Appearance > Slow mode

This will hide timeline updates behind a click instead of automatically scrolling the feed

**Reply by and-rea** [reply]
Good one! Didn't understand at first read the effect of that setting. Still I don't understand that phrasing but it works xD

I had preferred a "Stop automated feeding of your timeline." probably xD

Thanks for pointing me to the setting!

**Comment by and-rea** [comment]
Update: it seems the fetching continuing anyway even with the setting active. Only visualization is stopped but network pulling doesn't.
![Screenshot 2023-10-12 153746](https://github.com/mastodon/mastodon/assets/13545311/a4680122-ab75-43f6-9515-7167690e28f1)

**Reply by StrangeMinds1960** [reply]
I don't see how an account could tell all instances it's federated with to stop sending new posts until it's ready.  How would other instances keep track of which posts you (as an individual) have already received?

**Reply by and-rea** [reply]
As I am the client, why the need to "tell" things to instances? I lost you in this.

I only must stop asking my server to feed me the new incoming objects, because I am scrolling to the past and it means I'm not interested in them, as I am actually searching the older posts.

In fact, scrolling down (older posts) on the page, when it reaches the bottom it asks the server for a number of older posts to show there.
The same may apply on top too. If I decide to scroll up, when I reach the top, the code on the page can ask a number of newer posts up the time it was at and show them. Right now, I see everything new is fed to the page regardless of my actual position in the timeline.
